### ISSUE TITLE
TODOs

### ISSUE THREAD

**Issue by rossbar** [rossbar/segment_anything_sandbox#1]
- [ ] Try with gpu: what framerates can we get for predictions
   - [ ] Modify interactive cursor to predict on hover, and accept when prediction is good
 - [ ] Iterative selection with logits masks
 - [ ] Sequential point prediction instead of batch w/ concatenated outputs
